Lib/uuid.py

```diff
@@ -234,7 +265,7 @@ def __init__(self, hex=None, bytes=None, bytes_le=None, fields=None,
             if len(fields) != 6:
                 raise ValueError('fields is not a 6-tuple')
             (time_low, time_mid, time_hi_version,
-             clock_seq_hi_variant, clock_seq_low, node) = fields
+                clock_seq_hi_variant, clock_seq_low, node) = fields
```

Comment:
Can we avoid cosmetic changes in this PR?

Reply:
Of course! This wasn't intentional, good find